O/OpenCL/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "OpenCL"
```

Comment:
Shouldn't this be called 
```suggestion
name = "OpenCL_ICD_Loader"
```
to match the project name?

Reply:
I thought about it. Do you think end-users will get lost with this weird name they chosen for the main project? For example, it took a while for me to figure out that this is the actual core OpenCL library. If users find OpenCL_ICD_Loader_jll in the registry they won't be sure if this is what they need. That is why I thought the simpler OpenCL_jll would be best for users.

Reply:
Let me know if it is ok to go ahead with OpenCL_jll only. I think the combo OpenCL_Headers_jll and OpenCL_jll reads better, and will be simpler to understand for end users.